templates/prescription/general_list.html

```diff
@@ -83,6 +83,28 @@
     }
 }
 
+function deleteDrug(d) {
+        let msg = 'Do you really want to delete ';
```

Comment:
For 5.0.3 you can use
let msg = xl('Do you really want to delete ');
I'm unsure if utility.js is in scope for this smarty but you can try.
Or:
let isPromise = top.jsFetchGlobals('e_rx');
        isPromise.then(msg => {
            alert(msg.e_rx.deleteWarn');
            .... add rest of your delete ajax function within resolved promise
        });
Then add your translation in library/ajax/phpvars_to_js.php. (this will also work in v5.0.2) but best choice is the new JS xl() function for 5.0.3 even if you have to include utility.js to forms scope.(note that utility.js is included automatic if the Header class is used.)